scripts/hab3_bench/bench_runner.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# NOTE: run this script from habitat-lab/ directory
+# TO PLOT RESULTS SEE RUN `python scripts/hab2_bench/plot_bench.py`
+mkdir -p data/profile
+NUM_STEPS=200
+set -e
+
+export OMP_NUM_THREADS=2
+export MAGNUM_LOG=quiet
+export HABITAT_SIM_LOG=quiet
+
+USE_ORACLE_ACTION=
+#NOTE: this creates a new URDF with no accompanying ao_config to avoid skinning
+cp data/humanoids/humanoid_data/female2_0.urdf data/humanoids/humanoid_data/female2_0_no_skinning.urdf
+NO_SKINNING="habitat.simulator.agents.agent_1.articulated_agent_urdf='data/humanoids/humanoid_data/female2_0_no_skinning.urdf'"
+
+#number of processes
+# shellcheck disable=SC2043
+for j in 1
```

Comment:
Add back the multi-process bench
```suggestion
for j in 1 16
```